packages/graphql-language-service/src/parser/api.ts

```diff
@@ -176,14 +176,15 @@ export function getContextAtPosition(
   schema: GraphQLSchema,
   contextToken?: ContextToken,
   options?: { mode?: GraphQLDocumentMode; uri?: string },
+  offset = 0,
```

Comment:
Just to explain this change: `getContextAtPosition` is used in 2 places:

In `getHoverInformation`, the offset needs to be changed to 0 to fix the off-by-one problem on hover.
https://github.com/graphql/graphiql/blob/b495159382465a2cab6d8592b8bce66fece066fe/packages/graphql-language-service/src/interface/getHoverInformation.ts#L39


In `getAutocompleteSuggestions`, the offset needs to be kept at 1 as the current behavior is correct.
https://github.com/graphql/graphiql/blob/b7e0a66af7ddc2cac30c943e55963a3e9745f376/packages/graphql-language-service/src/interface/getAutocompleteSuggestions.ts#L145-L152